app/build.gradle

```diff
@@ -89,8 +93,6 @@ dependencies {
     implementation 'com.jakewharton.timber:timber:4.7.1'
 
     implementation 'com.squareup.okhttp3:okhttp:3.12.6'
-    implementation "com.jakewharton:butterknife:$butterknife_version"
-    annotationProcessor "com.jakewharton:butterknife-compiler:$butterknife_version"
```

Comment:
I guess I'm late for the party, but the definition of `butterknife_version` should be removed along with these lines imo: 

https://github.com/criticalmaps/criticalmaps-android/blob/master/build.gradle#L4

Reply:
Ah, missed that - thanks!